src/common.h

```diff
@@ -86,8 +86,11 @@
  * Change the visibility of interface functions
  */
 
-#define LC3_EXPORT \
-    __attribute__((visibility ("default")))
+#ifdef _WIN32
```

Comment:
The bit in https://github.com/google/liblc3/pull/49#issuecomment-2008282801 as regards static libraries on Windows still needs addressing.